plugins/catalog-backend/src/service/CatalogPlugin.ts

```diff
@@ -28,18 +28,21 @@ import {
 } from '@backstage/plugin-catalog-node/alpha';
 import {
   CatalogProcessor,
+  CatalogProcessorParser,
   EntityProvider,
   ScmLocationAnalyzer,
 } from '@backstage/plugin-catalog-node';
 import { loggerToWinstonLogger } from '@backstage/backend-common';
 import { PlaceholderResolver } from '../modules';
+import { defaultEntityDataParser } from '../modules/util/parse';
 
 class CatalogProcessingExtensionPointImpl
   implements CatalogProcessingExtensionPoint
 {
   #processors = new Array<CatalogProcessor>();
   #entityProviders = new Array<EntityProvider>();
   #placeholderResolvers: Record<string, PlaceholderResolver> = {};
+  entityDataParser?: CatalogProcessorParser;
```

Comment:
```suggestion
  #entityDataParser?: CatalogProcessorParser;
```
I believe the pattern we're trying to stick with is private internal variables for this extension point, and `#` prefix enforces that.